src/main/java/codeu/model/store/basic/MessageStore.java

```diff
@@ -65,6 +65,12 @@ private MessageStore(PersistentStorageAgent persistentStorageAgent) {
     messages = new ArrayList<>();
   }
 
+  /** Access the current set of messages known to the application. This function 
+  is temporary for the Activiy Feed MVP. */
```

Comment:
Do we have a plan for removing this method? If so, how about adding it in a TODO here instead of the part about its being temporary?

Reply:
I still don't have a specific plan but I'll add a TODO since this approach would be inefficient with an Event class.